{"uuid": "e090553b-908c-414c-80f9-f75ee1e2b7bc", "children": ["dd1b7759-ede9-4649-a6a7-91eb2d24e0e2"], "befores": [{"name": "setup", "status": "passed", "start": 1709709030657, "stop": 1709709034659}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\benik\\PycharmProjects\\pythonAutomationStore\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\benik\\PycharmProjects\\pythonAutomationStore\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 926, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\benik\\PycharmProjects\\pythonAutomationStore\\tests\\conftest.py\", line 26, in setup\n    driver.quit()\n  File \"C:\\Users\\benik\\PycharmProjects\\pythonAutomationStore\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 188, in quit\n    self.service.stop()\n  File \"C:\\Users\\benik\\PycharmProjects\\pythonAutomationStore\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 146, in stop\n    self.send_remote_shutdown_command()\n  File \"C:\\Users\\benik\\PycharmProjects\\pythonAutomationStore\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 131, in send_remote_shutdown_command\n    if not self.is_connectable():\n           ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\benik\\PycharmProjects\\pythonAutomationStore\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 120, in is_connectable\n    return utils.is_connectable(self.port)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\benik\\PycharmProjects\\pythonAutomationStore\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 101, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Program Files\\Python312\\Lib\\socket.py\", line 844, in create_connection\n    exceptions.clear()  # raise only the last error\n    ^^^^^^^^^^^^^^^^^^\n"}, "start": 1709709048414, "stop": 1709709049599}], "start": 1709709030657, "stop": 1709709049599}